core/shared/src/main/scala/scalaz/zio/package.scala

```diff
@@ -26,6 +26,14 @@ package object zio extends EitherCompat {
   type Task[+A]   = ZIO[Any, Throwable, A]
   type UIO[+A]    = ZIO[Any, Nothing, A]
 
+  implicit class ZIOInvarant[R, E, A](val self: ZIO[R, E, A]) extends AnyVal {
```

Comment:
typo in `Invariant`

Also, I think that if you put it in the `ZIO` companion object, it won't need to be imported.

Reply:
Fixed both